st2common/st2common/bootstrap/runnersregistrar.py

```diff
@@ -134,8 +134,8 @@
                 'secret': True
             },
             'private_key': {
-                'description': ('Private key material to log in. Note: This needs to be actual '
-                                'private key data and NOT path.'),
+                'description': ('Private key material or path to the private key file on disk '
```

Comment:
Depending on what user the action runner runs as, this is a huge problem without documentation. If the action runner process runs as ``st2``, st2 needs to be able to read this key. There should be docs around the risk of making the key readable by st2. Without a corrresponding change to deprecate core.local, this IMO is a huge issue. I am -1 to supporting path to key file.